tests/test_ingest.py

```diff
@@ -41,8 +41,32 @@ class DecamIngestTestCase(IngestTestBase, lsst.utils.tests.TestCase):
     def setUp(self):
         self.ingestDir = os.path.dirname(__file__)
         self.instrument = lsst.obs.decam.DarkEnergyCamera()
+        # DecamRawIngestTask ingests every detector in each raw file, so we
+        # only have to specify one file here, but should get two dataIds
+        # in the output repo.
         self.file = os.path.join(testDataDirectory, "rawData", "raw", "raw.fits")
-        self.dataId = dict(instrument="DECam", exposure=229388, detector=25)
+        self.dataIds = [dict(instrument="DECam", exposure=229388, detector=25),
+                        dict(instrument="DECam", exposure=229388, detector=1)]
+        self.RawIngestTask = lsst.obs.decam.DecamRawIngestTask
+
+        super().setUp()
+
+
+class DecamIngestFullFileTestCase(IngestTestBase, lsst.utils.tests.TestCase):
+    """Test ingesting a file that contains all "normal" DECam HDUs.
+    """
+    def setUp(self):
+        self.ingestDir = os.path.dirname(__file__)
+        self.instrument = lsst.obs.decam.DarkEnergyCamera()
+        # DecamRawIngestTask ingests every detector in each raw file, so we
+        # only have to specify one file here, but should get many dataIds
+        # in the output repo.
+        self.file = os.path.join(testDataDirectory, "rawData", "raw",
+                                 "c4d_150227_012718_ori-stripped.fits.fz")
+        self.dataIds = []
+        for i in range(1, 63):
+            self.dataIds.append(dict(instrument="DECam", exposure=415282, detector=i))
```

Comment:
consider the list comprehension form:
```python
self.dataIds = [{"instrument": "DECam", "exposure": 415282, "detector": i} for i in range(1, 63)]
```

Reply:
Ah, that fits in 110 chars. I'd tried a different list comp that was like 120. Thanks.